src/lib/legacy.ts

```diff
@@ -309,8 +309,18 @@ export async function fromV2TargetApps(
 						const appId = parseInt(id, 10);
 						const app = apps[appId];
 
-						// If local mode just use id as uuid
-						const uuid = local ? id : await getUUIDFromAPI(appId);
+						let uuid: string;
+						try {
+							// If local mode just use id as uuid
+							uuid = local ? id : await getUUIDFromAPI(appId);
```

Comment:
How about using`uuid = local ? id : await getUUIDFromAPI(appId).catch(() => id);` to avoid the `let` and wrapping into a `try ... catch`?